infrastructure/modules/aws/efs/variables.tf

```diff
@@ -0,0 +1,34 @@
+# Tags
+variable "tags" {
+  description = "Tags for resource"
+  type        = any
+  default     = {}
+}
+
+# VPC info
+variable "vpc" {
+  description = "AWS VPC info"
+  type = object({
+    id          = string
+    cidr_blocks = list(string)
+    subnet_ids  = list(string)
+  })
+}
+
+# EFS info
+variable "efs" {
+  description = "EFS info"
+  type = object({
+    name                            = string
+    kms_key_id                      = string
+    performance_mode                = string # "generalPurpose" or "maxIO"
+    throughput_mode                 = string #  "bursting" or "provisioned"
+    provisioned_throughput_in_mibps = number
+    transition_to_ia                = string # "AFTER_7_DAYS", "AFTER_14_DAYS", "AFTER_30_DAYS", "AFTER_60_DAYS", or "AFTER_90_DAYS"
```

Comment:
Couldn't you add this to validation? (I think you can have multiple validation blocks)

Reply:
done